site/en/blog/optimizing-lcp-using-signed-exchanges/index.md

```diff
@@ -0,0 +1,459 @@
+logData 0
+
+// Visit a search result page that includes your page.
+navigate https://google.com/search?q=site%3Asigned-exchange-testing.dev+image
+
+// Wait for the prefetch to succeed.
+sleep 10
+
+// Re-enable log collection.
+logData 1
+
+// Navigate to the prefetched SXG on the Google SXG Cache.
+navigate https://signed--exchange--testing-dev.webpkgcache.com/doc/-/s/signed-exchange-testing.dev/sxgs/valid-image-subresource.html
+```
+
+If your page doesn't appear in any Google Search results yet, you can use [this prefetch page](https://signed-exchange-testing.dev/prefetch/) to generate a pretend search results page for this purpose.
+
+
+{% Aside %}
+Note that when Google Search instructs a user's browser to prefetch your SXG, it does so via a URL on webpkgcache.com in order to [avoid revealing the user's intent](https://wicg.github.io/webpackage/draft-yasskin-wpack-use-cases.html#name-privacy-preserving-prefetch) before they navigate. These URLs don't affect the display or behavior of the page, because the browser trusts the original URL that has been signed into the SXG.
+{% endAside %}
+
+The easiest way to determine the webpkgcache.com URL is to visit your page using the [SXG Validator Chrome extension](https://chrome.google.com/webstore/detail/sxg-validator/hiijcdgcphjeljafieaejfhodfbpmgoe), and click the extension icon to see the cache URL:
+
+{% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/DbkcylGpILjOzOnxREXl.png",
+       alt="SXG Validator showing cache information including URL",
+       width="506", height="229" %}
+
+Once these tests are complete, go to [Test History](https://app.webpagetest.org/ui/wpt/testhistory), select the two tests, and click Compare:
+
+{% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/ElSV7nAWCWV0Yjf7VC2o.png",
+       alt="Test History with two tests checked and the Compare button highlighted",
+       width="509", height="202" %}
+
+Append `?medianMetric=LCP` to the compare URL so that the run with median LCP is selected for each side of the comparison. (The default is median by Speed Index.)
+
+To compare waterfalls, expand the Waterfall Opacity section and drag the slider. To view the video, click Adjust Filmstrip Settings, scroll down inside that dialog, and click View Video.
+
+## Debug
+
+If the WebPageTest is showing that the HTML is being prefetched, then it has succeeded in all the steps of the pipeline; you may skip to the [Optimize](#optimize) section to learn how to further improve LCP. Otherwise, you'll need to find out where in the pipeline it failed and why; see the following sections for how.
+
+### Publishing
+
+Make sure your pages are being generated as signed exchanges.
+
+{% Aside %}
+Your site will not serve SXG to users, even on SXG-capable browsers. They're meant for SXG-capable crawlers, to enable prefetch. (Serving SXGs to users who visit the site directly only adds overhead over a perfectly cromulent web page.) Thus, to debug, you need to pretend to be a crawler.
```

Comment:
I missed this during edit: I get the reference with the word "cromulent"—and I love that episode, btw 😂—but our content is consumed by an international audience, and with "cromulent" being a made-up word from a primarily English-language television show, I wonder if it won't just cause confusion. I'd consider changing this to "perfectly acceptable" or some such equivalent.

Reply:
Thanks! Good suggestion; fixed.